src/framework/linalg/almost_equal.hpp

```diff
@@ -19,29 +19,109 @@
 #include <limits>
 #include <type_traits>
 
+#include "framework/matrix.hpp"
 #include "framework/linalg/enable_if_numeric.hpp"
 
+#define MAXDIFF 5.0*AER::Linalg::epsilon<T>::value
```

Comment:
What are the reason for these (and the 5.0 * factor)?

Reply:
These were introduced in https://github.com/Qiskit/qiskit-aer/commit/3589fe9b0d76076d794a2fce8de559a4c1607a53.
It seems they are used as default values for almost_equal functions but now with the 5.0 factor. I'm not sure if the 5.0 factor was introduced for some reason. Maybe @ereastman can shed some light?